mne/viz/backends/_abstract.py

```diff
@@ -532,7 +532,9 @@ def _dock_add_label(self, value, align=False, layout=None):
         pass
 
     @abstractmethod
-    def _dock_add_button(self, name, callback, tooltip=None, layout=None):
+    def _dock_add_button(
+        self, name, callback, style='pushbutton', tooltip=None, layout=None
```

Comment:
If you're going to add to the middle, let's be safe
```suggestion
        self, name, callback, *, style='pushbutton', tooltip=None, layout=None
```

Reply:
Exactly what I meant 👍